src/main/java/com/hiqes/andele/RequestOwnerAppCompatActivity.java

```diff
@@ -21,7 +21,7 @@
 import android.content.Context;
 import android.content.pm.PackageManager;
 import android.os.Build;
-import android.support.v7.app.AppCompatActivity;
+import androidx.appcompat.app.AppCompatActivity;
```

Comment:
Same comment as above regarding the fragments.  We should support both `android.support` and `androix.appcompat`.

Reply:
Both are now supported